.nuget/create_nuget.py

```diff
@@ -36,9 +40,44 @@
         <dependencies> </dependencies>
     </metadata>
         <files>
+        <file src="build\**" target="build"/>
         <file src="runtimes\**" target="runtimes"/>
         </files>
     </package>
     ''' % version
+    prop_str = '''
+    <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup Condition="Exists('packages.config') OR
+                            Exists('$(MSBuildProjectName).packages.config') OR
+                            Exists('packages.$(MSBuildProjectName).config')">
+        <Content Include="$(MSBuildThisFileDirectory)\..\runtimes\win-x64\native\*.dll"
```

Comment:
This doesn't work because `\r` and `\n` are getting treated as carriage return and newline characters.

I would check the `.props` and `.targets` files into the repo, and treat them like any normal file.